reference-architecture/rhv-ansible/example/ocp-vars.yaml.36

```diff
@@ -0,0 +1,92 @@
+---
```

Comment:
I don't really know because I didn't diff'ed them but if the only difference is the ocp version variable, maybe you can use a single variable file and specify that variable needs to be changed.

Reply:
That's not the only difference... Besides, this file is the place where all the variables are to be set.